app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container/operations.rb

```diff
@@ -0,0 +1,15 @@
+module ManageIQ::Providers::Amazon::CloudManager::CloudObjectStoreContainer::Operations
+  extend ActiveSupport::Concern
+
+  def raw_destroy
+    raise "CloudObjectStoreContainer has no #{ui_lookup(:table => "ext_management_systems")}, unable to destroy it" unless ext_management_system
+    with_provider_object(&:delete!)
+    destroy
```

Comment:
You are referring to the policy checks to prevent actions like: [app/models/vm/operations/guest.rb#L23](https://github.com/ManageIQ/manageiq/blob/1261a4636668854e446fe1e603c8cc4bc4a6e626/app/models/vm/operations/guest.rb#L23)

That would be a good idea, but it should be in base method that calls the `raw_*` method, like the example above.